build/dockerfiles/content_sets_centos8.repo

```diff
@@ -0,0 +1,11 @@
+[centos-8-for-appstream]
```

Comment:
as you're moving files, maybe we should move specific files for rhel.Dockerfile to `rhel` folder ?